### PR TITLE
Add agenda for 2023-10-25 call on Intl.MessageFormat

### PR DESCRIPTION
**Pull request by eemeli**
Adding as a draft until the dates are settled via the [Doodle poll](https://doodle.com/meeting/organize/id/elp2Bnre).

The proposed agenda is [here](https://github.com/eemeli/tc39-incubator-agendas/blob/add-intl-messageformat/2023/10-25.md).

I'm hoping to be able to pick the meeting date & time by the end of this week.

### CODE REVIEW

**Review by syg — APPROVED** [review]

**Comment by syg** [comment]
This is marked as draft, so I can't merge it. Was this intentional?

**Comment by ljharb** [comment]
> Adding as a draft until the dates are settled

File: 2023/10-xx.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Agenda for Ecma TC39 Incubator Call
+
+- **Date & Time**: TBD, see [Doodle poll](https://doodle.com/meeting/participate/id/elp2Bnre)
```
Comment (eemeli) [inline 2023/10-xx.md]:
```suggestion
- **Date**: 25 Oct, 2023
- **Time**: 9:00 PDT / 18:00 CEST
```

**Review by ljharb — APPROVED** [review]

File: 2023/10-25.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Agenda for Ecma TC39 Incubator Call
+
+- **Date**: 25 Oct, 2023
+- **Time**: 9:00 PDT / 18:00 CEST
+- **Attendee information**: [Reflector](https://github.com/tc39/Reflector/issues/508)
+- **Call** TBD
```
Comment (eemeli) [inline 2023/10-25.md]:
```suggestion
- **Call** [Google Meet](https://meet.google.com/snx-nfpo-ehs)
```

**Pull request merged**